typespec-tests/src/main/java/com/cadl/enumservice/EnumServiceAsyncClient.java

```diff
@@ -756,7 +756,7 @@ public Mono<String> setStringEnumArray(List<ColorModel> colorArray, List<ColorMo
         }
         return setStringEnumArrayWithResponse(colorArray.stream()
             .map(paramItemValue -> Objects.toString(paramItemValue, "")).collect(Collectors.toList()), requestOptions)
-                .flatMap(FluxUtil::toMono).map(protocolMethodData -> protocolMethodData.toObject(String.class));
+            .flatMap(FluxUtil::toMono).map(protocolMethodData -> protocolMethodData.toObject(String.class));
```

Comment:
There is some small diff. Not sure what's the cause. Maybe some version diff of dependency lib?

We can merge and hope SDK repo checkstyle still happy with it.

Report here should give me an early warning.
https://github.com/Azure/autorest.java/wiki/Fluent-Codegen-Report

Reply:
Yeah, this small formatting is something I'll keep an eye on whether we see skew between this change and running with Spotless once I begin Spotless integration in azure-sdk-for-java